redirect/routes.php

```diff
@@ -329,6 +331,17 @@ public function getTargetOfSub($host) {
         if (isset($this->synonyms[$redirectUrl])) {
             $redirectUrl = $this->synonyms[$redirectUrl];
         }
+        //moodle support; example: mgad.tum.sexy
+        elseif (strlen($redirectUrl) > 1 && substr($redirectUrl, 0, 1) === 'm' && !isset($this->routes[$redirectUrl])){
+            $tmp = substr($redirectUrl, 1);
+            if (isset($this->synonyms[$tmp])) {
+                $redirectUrl = $this->synonyms[$tmp];
+            }
+            else {
+                $redirectUrl = $tmp;
```

Comment:
You can save this else branch, if you directly assign to `$redirectUrl`. If there is a synonym, you will overwrite it anyway.